src/pages/messages/components/MessageList.tsx

```diff
@@ -73,53 +87,34 @@ export const MessageItem = ({ text, date, username, backgroundColor, onDelete, i
         bgcolor: backgroundColor,
         p: '0.5rem',
         borderRadius: '4px',
-        cursor: onDelete && !expanded ? 'pointer' : 'auto',
         display: 'flex',
         flexDirection: 'column',
-      }}
-      onClick={() => setExpanded(true)}>
-      <Typography sx={{ display: 'flex', flexWrap: 'wrap', justifyContent: 'space-between' }}>
-        <span>
-          {senderQuery.isPending ? (
-            <Skeleton sx={{ width: '8rem' }} />
-          ) : (
-            <b data-testid={dataTestId.registrationLandingPage.tasksPanel.messageSender}>
-              {senderName ? senderName : <i>{t('common.unknown')}</i>}
-            </b>
-          )}
-        </span>
-        <span data-testid={dataTestId.registrationLandingPage.tasksPanel.messageTimestamp}>{toDateString(date)}</span>
-      </Typography>
+      }}>
+      <Box sx={{ display: 'grid', gridTemplateColumns: '1fr auto', alignItems: 'center' }}>
+        <Typography sx={{ display: 'flex', flexWrap: 'wrap', justifyContent: 'space-between' }}>
+          <Tooltip title={senderName ? senderName : t('common.unknown')}>
+            <span style={{ marginRight: '0.5rem' }}>
```

Comment:
Oppbygninga på dette virker litt snedig nå (det var det før denne PRen også skal sies). Vi har bl.a. en Typography som bare står for styling med flex 🤔 
Er ikke det vi egentlig vil ha mulig å få til på en langt enklere måte, ala dette (veldig forenklet, men riktig semantikk):

```
<Box sx={{ display: 'grid', gridTemplateColumns: '1fr auto auto', alignItems: 'center' }}>
  <Typography>{senderName ?? <i>{t('common.unknown')}</i>}</Typography>
  <Typography>{toDateString(date)}</Typography>
  {menuElement}
</Box>
```